addons/mass_mailing/static/src/js/mass_mailing_wysiwyg.js

```diff
@@ -193,14 +193,28 @@ export class MassMailingWysiwyg extends Wysiwyg {
         if (isWithinBackgroundImage) {
             this.toolbarEl.querySelector('#create-link').classList.toggle('d-none', true);
         }
+        //remove checklist command for mass_mailing
+        const checklistCommand = this.odooEditor.powerbox.commands.find(
+            command => command.fontawesome === "fa-check-square-o"
+        );
+        if (checklistCommand) {
+            checklistCommand.isDisabled = () => true;
+        }
```

Comment:
This seems a bit meh, can't we pass down "disableCheckbox" through the config or something, like you did above?